src/main/java/org/gradle/profiler/studio/StudioGradleClient.java

```diff
@@ -103,8 +105,9 @@ public void close() {
             System.out.println("* Android Studio stopped.");
         } catch (Exception e) {
             e.printStackTrace();
-            System.out.println("* Android Studio did not stopped successfully, you will have to close it manually.");
+            System.out.println("* Android Studio did not finish successfully, you will have to close it manually.");
         } finally {
+            tryClose(studioPluginInstaller::uninstallPlugin);
```

Comment:
What arcane magic is this? You are "auto-boxing" that method reference in a `Closable` and executing it as via `tryClose()` to..? Why?! :)

I never knew you could do this... I'm not sure how I feel. Conflicted..?!

Reply:
Fixed that with try-with-resources. That solution was not nice, I agree